### ISSUE TITLE
css bug!

### ISSUE THREAD

**Issue by xgqfrms-gildata** [drublic/css-modal#218]
# css bug!

> sass in css bug.

![image](https://user-images.githubusercontent.com/29531779/36955565-2c741a22-2064-11e8-85e1-264d6a950a44.png)

```css

/* Header */


/*
.css-modal_header {
    border-bottom: 1px solid var(--modal--border-color);
    padding: 0 1.2em;
    >h2 {
        margin: 0.5em 0;
    }
}
*/

.css-modal_header {
    border-bottom: 1px solid var(--modal--border-color);
    padding: 0 1.2em;
}

.css-modal_header>h2 {
    border-bottom: 1px solid var(--modal--border-color);
    padding: 0 1.2em;
    margin: 0.5em 0;
}

```

**Comment by xgqfrms-gildata** [comment]
# api demo too confused me!

> I just make it more simple, more clear.

```html

```

**Comment by drublic** [comment]
@xgqfrms-gildata Thanks for the report. Fixed it in ecce4ea.

I get the other problem. I find it too hard too and will try to improve with the new version.
